moysklad/moysklad_class_lib.py

```diff
@@ -0,0 +1,206 @@
+""" В модуле хранится описание классов.
+
+"""
+import requests
+import os, sys
+import logging
+import logging.config
+import logger_config
+import project_settings
+
+sys.path.insert(1, project_settings.PROJECT_PATH)
+
+import moysklad.moysklad_privatedata as ms_pvdata
+import moysklad.moysklad_lib as ms_lib
+import moysklad.moysklad_urls as ms_urls
+
+
+
+class MoySklad(object):
+    """ Класс описывает работу с сервисом МойСклад"""
+    token = ''  # токен для работы с сервисом
+    logger = None
+
+    def __init__(self):
+        try:
+            self.token = ms_lib.get_access_token(ms_pvdata.USER, ms_pvdata.PASSWORD)
+            logging.config.dictConfig(logger_config.LOGGING_CONF)
+            self.logger = logging.getLogger("moysklad")
+        except Exception as error:
+            logger.exception(f"Не удалось создать инстанс MoySklad")
+
+
+    def get_goods_compliance_egais(self, sold_goods: list, comp_table: list):
+        """
+        Метод сранивает два списка sold_goods и comp_table, возвращает новый sold_goods, c заполнеными наименованиями ЕГАИС
+        :param sold_goods:
+        :param comp_table:
+        :return:
+        """
+        # sold_goods - отсортированый список кортежей
+        # [(Наименование, Количество, Цена), (Наименование, Количество, Цена), ...]
+        # comp_table - отсортированый список списков
+        # [[Наименование, Наименование ЕГАИС], [Наименование, Наименование ЕГАИС], ...]
+        if len(sold_goods) == 0 or len(comp_table) == 0:
```

Comment:
Не проверяй длинну, проверяй просто булеаном
```suggestion
        if not sold_goods or not comp_table:
```

Reply:
ok